tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php

```diff
@@ -1272,4 +1272,19 @@ public function provideInvalidPhpdocCases()
             ],
         ];
     }
+
+    public function testTypesContainingWhitespace()
```

Comment:
Could you add another test for something like `callable(string): bool`?

Reply:
Example from the wild: https://github.com/php-http/client-common/blob/master/src/Plugin.php#L27.

Reply:
ping @kubawerlos

Reply:
@enumag the support for callables is here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4914 - after this is merged I will continue with this PR.